README.md

```diff
@@ -17,29 +17,29 @@ PanelPal is a python package of command line tools for helping UK labs implement
 
 To set up a conda environment for this project, you can use the provided `environment.yaml` file.
 
-1. Clone or download this repository.
+1. Clone or download this repository:
 
    ```bash
    git clone https://github.com/PatrickWeller/PanelPal.git
     ```
 
-2. Create a new conda environment with the following command:
+2. Build the docker image:
 
    ```bash
-   conda env create -f env/environment.yaml
+   cd PanelPal
+   docker build -it panelpal .
```

Comment:
typo, should be:
docker build -t panelpal .

Reply:
Corrected in latest commit